stdlib/email/headerregistry.pyi

```diff
@@ -44,6 +52,8 @@ class DateHeader:
 class UniqueDateHeader(DateHeader): ...
```

Comment:
This should be chagned just like all other `UniqueFoo` classes.

Reply:
Thanks!